### PR TITLE
Add hero styles

### PR DESCRIPTION
**Pull request by Don-Stevenson**
Switching over hero styles from bootstrap to style components

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,34 +1,35 @@
 import React, { useState } from "react"
 import { BrowserRouter as Router, Route, Link } from "react-router-dom"
-import styled from "styled-components";
```
Comment (PandaComp) [inline src/App.js:3]: I love semi colons!

Comment (Don-Stevenson) [reply src/App.js:3]: Too bad!

Comment (PandaComp) [reply src/App.js:3]: You dick!

File: src/components/Hero/index.js [hunk 2]
```diff
@@ -1,28 +1,19 @@
 import React from "react"
-import styled from "styled-components";
-
```
Comment (PandaComp) [inline src/components/Hero/index.js:3]: I don't know what's going on. Comment pls

Comment (PandaComp) [reply src/components/Hero/index.js:3]: Cheers buddy!!!

**Review by PandaComp — APPROVED** [review]
You killed it, broski!!

**Pull request merged**